db/config.cc

```diff
@@ -737,6 +738,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
         "\tcertificate: (Default: conf/scylla.crt) The location of a PEM-encoded x509 certificate used to identify and encrypt the client/server communication.\n"
         "\tkeyfile: (Default: conf/scylla.key) PEM Key file associated with certificate.\n"
         "truststore : (Default: <system truststore> ) Location of the truststore containing the trusted certificate for authenticating remote servers.\n"
+        "certficate_revocation_list : (Default: <empty> ) PEM encoded certificate revocation list.\n"
```

Comment:
There's another copy of this in `alternator_encryption_options`. If I understand correctly, if you add it there too, it will just work?

Reply:
Since alternator is purely a listen-connection, without certificate authorization, the option is useless for it. 
Only the initiating side of a TLS connection verifies certificates (unless again you use certificate auth - like client connection does when `require_client_auth` is true).

Reply:
Good point.